tests/rptest/tests/tx_admin_api_test.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.mark.resource import cluster
+from rptest.clients.types import TopicSpec
+from rptest.services.admin import Admin
+
+import confluent_kafka as ck
+
+import random
+
+from rptest.tests.redpanda_test import RedpandaTest
+
+
+class TxAdminTest(RedpandaTest):
+    topics = (TopicSpec(name="tx_test",
+                        partition_count=1,
+                        replication_factor=3), )
+
+    def __init__(self, test_context):
+        super(TxAdminTest, self).__init__(test_context=test_context,
+                                          num_brokers=3,
+                                          extra_rp_conf={
+                                              "enable_idempotence": True,
+                                              "enable_transactions": True
+                                          })
+
+        self.admin = Admin(self.redpanda)
+
+    @cluster(num_nodes=3)
+    def test_simple_get_transaction(self):
+        self.producer1 = ck.Producer({
+            'bootstrap.servers':
+            self.redpanda.brokers(),
+            'transactional.id':
+            '0',
+        })
+        self.producer2 = ck.Producer({
+            'bootstrap.servers':
+            self.redpanda.brokers(),
+            'transactional.id':
+            '1',
+        })
+        self.producer1.init_transactions()
+        self.producer2.init_transactions()
+        self.producer1.begin_transaction()
+        self.producer2.begin_transaction()
+
+        self.producer1.produce(self.topics[0].name, '0', '0')
+        self.producer2.produce(self.topics[0].name, '0', '1')
+
+        self.producer1.flush()
+        self.producer2.flush()
+
+        txs_info = self.admin.get_transactions(self.topics[0].name, 0)
+        assert (len(txs_info["active_transactions"]) == 2)
+        assert (txs_info["active_transactions"][0]['producer_id']['id'] == 1)
```

Comment:
we should not do assumption about value of the producer_id, let's just check that they are not equal